### DISCUSSION TITLE
Issue with Google

### DISCUSSION THREAD

**Discussion by themcgarvie** [abacritt/angularx-social-login#514] [General]
I am using the demo code in the repo. I have created a Google API end point and replaced demo with mine likewise for FB.

I did install using: @abacritt/angularx-social-login

It works for FB fine, but when I sign in via Google I get this:

ERROR Error: Uncaught (in promise): Object: {"error":"popup_closed_by_user"}

**Comment by Heatmanofurioso** [comment]
Hey. `@abacritt/angularx-social-login:1.2.1` has been deployed, can you test it with the new version if the lib, please?
